covins_backend/src/matcher/opengv/FrameNoncentralRelativeAdapter.cpp

```diff
@@ -0,0 +1,176 @@
+/*
+ * FrameNoncentralRelativeAdapter.cpp
+ * @brief Adapter for Non-central relative pose RANSAC (2D-2D)
+ * @author: Manthan Patel
+ * Created on: Apr 12,2022
+ */
+
+
+
+#include "covins/matcher/opengv/rel_pose/FrameNoncentralRelativeAdapter.hpp"
+
+namespace opengv {
+
+namespace relative_pose {
+
+FrameNoncentralRelativeAdapter::FrameNoncentralRelativeAdapter(
```

Comment:
Is this function really needed, isn't it easier to just use the generic one for all cases?

Reply:
I am not very sure but as far as I remember, the generic function will break for the case when there is only 1 KF in a set. It works only for the cases of 2+ KF and hence we can keep it for now.